File: data-structures/arrays/static/cpp/staticArray.cpp
#include <iostream>
using namespace std;


int main() {
    const int SIZE = 5;
    int arr[SIZE] = {1, 2, 3, 4, 5};

    cout << "array 'arr': " << endl;
    cout << "----------------------------" << endl;
    for (int i = 0; i < SIZE; ++i) {
        cout << "element at index " << i << ": " << arr[i] << endl;
    }
}

File: data-structures/arrays/static/java/staticArray.java

import java.util.random.RandomGenerator;

public class staticArray {
    
    public static void main(String[] args) {
        int[] arr = new int[10];
        for (int i = 0; i < arr.length; i++) {
            arr[i] = RandomGenerator.getDefault().nextInt(1, 10);  // populating the array with random integers.
        }

        System.out.println("""
                           array 'arr'elements:  
                           --------------------""");
        for (int i = 0; i < arr.length; i++) {
            System.out.println("element #" + (i + 1) + ": \t" + arr[i]);
        }

    }
}